middleware.js

```diff
@@ -28,65 +33,68 @@ var imports = {};
  * Following that we have a `staticProvider` layer setup to serve the .css
  * files generated by Sass.
  *
- *      var server = connect.createServer(
- *          sass.middleware({
- *              src: __dirname
- *            , dest: __dirname + '/public'
- *          })
- *        , connect.static(__dirname + '/public')
- *      );
+ *   var server = connect.createServer(
+ *     sass.middleware({
+ *       src:  __dirname,
+ *       dest: __dirname + '/public'
+ *     }),
+ *     connect.static(__dirname + '/public')
+ *   );
  *
  * @param {Object} options
  * @return {Function}
  * @api public
  */
 
-module.exports = function(options){
+module.exports = function(options) {
   options = options || {};
 
-  // Accept src/dest dir
-  if ('string' == typeof options) {
+  // Accept single src/dest dir
+  if (typeof options == 'string')
     options = { src: options };
-  }
-
-  // Force compilation
-  var force = options.force || options.response;
 
-  // Debug option
-  var debug = options.debug;
-
-  // Source dir required
-  var src = options.src;
-  if (!src) { throw new Error('sass.middleware() requires "src" directory'); }
-
-  // Default dest dir to source
+  // Source directory (required)
+  var src = options.src || function() {
+    throw new Error('sass.middleware() requires "src" directory.');
+  }();
+  // Destination directory (source by default)
```

Comment:
Sorry, it should be:

```javascript
var error = typeof options.error === 'function' ? options.error || function (err) {
  throw err;
}
```

because node-sass here returns `Error` object.